---
# The first string in the buffer is the current sentence
string[] sentence_buffer
